src/styles/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 0.3s, color 0.3s ease;
```

Comment:
Можна створити змінну time-transition(і в header теж використати її).
7. [SASS] - use variables for the main values so that you'll be able to reuse them and give them descriptive names.